### PR TITLE
redshift: use Python 3.5

### PR DESCRIPTION
**Pull request by pyhalov**

### CODE REVIEW

File: components/x11/redshift/Makefile [hunk 1]
```diff
@@ -32,11 +32,9 @@ COMPONENT_ARCHIVE_URL=	https://github.com/jonls/redshift/releases/download/v$(CO
 COMPONENT_LICENSE=	GPL-3
 COMPONENT_LICENSE_FILE=	COPYING
```
Comment (AndWac) [inline components/x11/redshift/Makefile:34]: TEST_TARGET=            $(NO_TESTS)
seems to be missing here.

Comment (Mno-hime) [reply components/x11/redshift/Makefile:34]: Added in 101b6a140dd59219a7f56526d339b1521fe53cba.

**Comment by pyhalov** [comment]
@Mno-hime Does it work for you? It says 
```
Trying location provider `geoclue2'...
Using provider `geoclue2'.
Using method `randr'.
Waiting for initial location to become available...
```

**Comment by pyhalov** [comment]
Do we need geoclue?

**Comment by pyhalov** [comment]
at least `redshift -l 47.233334:39.700001` works here...

**Comment by Mno-hime** [comment]
This works for me:
```
 newman  ~  cat .config/redshift/redshift.conf 
[redshift]
location-provider=manual

[manual]
lat=50.0855600
lon=14.3460961

;[randr]
;screen=0
```

**Review by Mno-hime — APPROVED** [review]
Works for me.

**Comment by Mno-hime** [comment]
> Do we need geoclue?

Possibly, I'll see if I can port it from solaris-userland and whether it works with Redshift.

**Comment by Mno-hime** [comment]
> > Do we need geoclue?
> 
> Possibly, I'll see if I can port it from solaris-userland and whether it works with Redshift.

Well, we don't need it _per se_, manual is fine. I ported geoclue (https://github.com/Mno-hime/oi-userland/tree/geoclue) but I can't make it work with Redshift. I think it's good the way it's now.

**Review by AndWac — APPROVED** [review]

**Pull request merged**